src/pages/lessons/[slug].tsx

```diff
@@ -218,7 +218,10 @@ const Lesson: FunctionComponent<LessonProps> = ({initialLesson}) => {
                   pip="true"
                   controls
                   onPlay={() => send('PLAY')}
-                  onPause={() => send('PAUSE')}
+                  onPause={() => {
+                    send('PAUSE')
+                    setIsPlaying(false)
```

Comment:
technically we should know "is playing" because the state is currently "playing" and there is no need for this extra `useState`

what do you think? why do we need `isPlaying` in addition to `playing` state?

Reply:
pushed the fixes